advanced/is-pattern-2/README.md

```diff
@@ -0,0 +1,285 @@
+# Helm Chart for a clustered deployment of WSO2 Identity Server
```

Comment:
@darsham3 I suppose the pattern name has to be `...Identity Server with Analytics Support`.